openrndr-core/src/main/kotlin/org/openrndr/shape/ShapeContour.kt

```diff
@@ -424,18 +472,18 @@ data class ShapeContour(val segments: List<Segment>, val closed: Boolean, val po
                     this
                 } else {
                     ShapeContour(
-                        segments.map { it.transform(transform) },
-                        closed,
-                        polarity
+                            segments.map { it.transform(transform) },
+                            closed,
+                            polarity
                     )
                 }
             }
 
     /**
-     * Sample a sub contour
-     * @param startT starting point in [0, 1)
-     * @param endT ending point in [0, 1)
-     * @return sub contour
+     * Returns a sample of the [ShapeContour].
```

Comment:
I left another comment about `sample`. Not sure if it can be worded differently to make it obvious that it returns a segment (but Segment has already meaning!) of the contour, a sub contour.